tests/platform_tests/api/test_chassis.py

```diff
@@ -487,6 +487,8 @@ def test_status_led(self, duthosts, enum_rand_one_per_hwsku_hostname, localhost,
             if status_led:
                 led_controllable = status_led.get("controllable", True)
                 led_supported_colors = status_led.get("colors")
+            else:
```

Comment:
why is 'status_led' missing in facts?

Reply:
I am not entirely sure -- the JSON file from which facts are gathered is probably outdated on our end. Here is the relevant part of the file -- no mention of status_led:

{
    "chassis": {
        "name": "Celestica-DX010-C32",
        "components": [
            {
                "name": "CPLD1"
            },
            {
                "name": "CPLD2"
            },
            {
                "name": "CPLD3"
            },
            {
                "name": "CPLD4"
            },
            {
                "name": "BIOS"
            }
        ],
        "fans": [

Reply:
@assrinivasan can you check with Celestica?